html-css/week1/jamshedali/assignment-week-1/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <!--header start -->
+    <header>
+        <h1>Order Summary</h1>
+        <div class="summary_total">
+            <div>Total</div>
+            <div>$333/month</div>
+        </div>
+        <div class="grand_total">
+            <div>Total</div>
+            <div>$100</div>
+        </div>
+    </header>
+   <!--header end -->
+    
+   <!-- main start  -->
+    <main>
+        <h1>Payment Method</h1>
```

Comment:
Avoid using multiple `<h1>` - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page
```suggestion
        <h2>Payment Method</h2>
```

Reply:
<h2>Payment Method</h2>  
and css 
main > h2{
  color: black;
  padding: 2%;
}
all h2 direct childs of main  (done)